apache2/msc_multipart.c

```diff
@@ -223,6 +260,17 @@ static int multipart_parse_content_disposition(modsec_rec *msr, char *c_d_value)
                 msr_log(msr, 9, "Multipart: Content-Disposition filename: %s",
                     log_escape_nq(msr->mp, value));
             }
+        } else if (strcmp(name, "filename*") == 0) {
+            if (filenameStar != NULL && strlen(filenameStar) != 0) {
```

Comment:
Should we allow "filename*=; ... ;filename*=UTF-8''fname"?
I don't think so.
We don't allow it for filename.

Reply:
I'm not sure I understand this question, because actually we don't allow this.

```
Content-Disposition: form-data; name="file"; filename*=; filename*=UTF-8''r%C3%A9sum%C3%A9.pdf
```

and I get:
```
ModSecurity: Multipart parsing error: Multipart: Invalid Content-Disposition header (-16): form-data; name="file"; filename*=; filename*=UTF-8''r%C3%A9sum%C3%A9.pdf. [hostname "localhost"] [uri "/post"] [unique_id "ZsyOgg3BHX5k_DYroV3swwAAAAA"]
ModSecurity: Warning. Match of "eq 0" against "REQBODY_ERROR" required. [file "/etc/modsecurity/modsecurity.conf"] [line "72"] [id "200002"] [msg "Failed to parse request body."] [data "Multipart parsing error: Multipart: Invalid Content-Disposition header (-16): form-data; name=\\x22file\\x22; filename*=; filename*=UTF-8''r%C3%A9sum%C3%A9.pdf."] [severity "CRITICAL"] [hostname "localhost"] [uri "/post"] [unique_id "ZsyOgg3BHX5k_DYroV3swwAAAAA"]
```